dapps/shop/backend/routes/shops.js

```diff
@@ -303,6 +319,20 @@ module.exports = function(app) {
         allFiles.push(file)
       }
       hash = String(allFiles[allFiles.length - 1].cid)
+    } else {
+      console.log(
+        'Shop not deployed to IPFS: Pinata not configured and not a dev environment.'
+      )
+    }
+    if (hash) {
```

Comment:
Might be worth creating this record even if we don't have an IPFS hash?  Maye to record an error?

Reply:
Yes, good point. Will be useful anyway to be able to audit the deployments. Will do.

Reply:
Done